### DISCUSSION TITLE
Welcome to playground-group-vrzn_appdev Discussions!

### DISCUSSION THREAD

**Discussion by go-christianharrel** [PUP-BSIT/playground-group-vrzn_appdev#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by go-christianharrel** [comment]
Here's a comment test inside discussions

**Comment by go-christianharrel** [comment]
Hello @PUP-BSIT/vrzn

**Comment by Galope-Jhean** [comment]
Hello World

**Reply by go-christianharrel** [reply]
LOW
